src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -1649,6 +1649,17 @@ hibernate.connection.url=jdbc:hsqldb:mem:rm;hsqldb.tx=mvcc;hsqldb.lob_file_scale
 
 To switch to a standalone database, see <<_perf_tuning_db>>.
 
+=== Tunning Linux for 15K nodes
+
+It is necessary to tune underlying Linux system, in case there will be plenty of nodes in Resource Manager.
+
+There should be big enough number of processes allowed, and number of open files as well.
```

Comment:
It is mandatory(or important) to allow having a large number of allowed (running?) processes and open files.

Reply:
it is mandatory to allow having a large number of running processes and open files.